pom.xml

```diff
@@ -208,30 +208,50 @@
                         <artifactId>maven-surefire-plugin</artifactId>
                         <version>2.22.0</version>
                         <configuration>
-                            <includes>
-                                <include>**/AllTestsSuite.java</include>
-                            </includes>
+                            <excludedGroups>slow</excludedGroups>
```

Comment:
"full" would seem to indicate that we're running everything. If we're not running slow tests in "full", where are we running them?

Reply:
The only test marked as slow is FrameEncryptionHandlerVeryLongTest, which takes 3 hours to run. I'm not sure under what circumstances we would want to run it, but you can always manually run it. Would it be better if I renamed this tag to something like manualInvocationOnly?

Reply:
Are we stuck with the name "full-test-suite"? Why not just "test-suite"?

Reply:
I've renamed it to test-suite. I've also renamed the tag to be "ad_hoc", as I felt it was a little confusing having a "slow" tag and also a fastTestsOnly system property.